cdap-ui/app/cdap/components/CustomDropdownMenu/index.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const CustomDropdownMenu = (props, context) => {
```

Comment:
why do we need a CustomDropdownMenu? if it's just the ability to change the Tag, reactstrap already has that capability, right?